util/fpga/cw_spiflash.py

```diff
@@ -0,0 +1,273 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+"""Utilities used to download firmware images into the OpenTitan system."""
+
+import time
+import hashlib
+from functools import partial
+
+
+def _uint_to_le(val, length):
```

Comment:
Is there a reason almost every symbol is _ prefixed...?

Reply:
Mostly because we tend to do `import foo` and call something from `foo` quite a bit in the ot-sca repo and things other than `SAM3UProgrammer` and `FtdiProgrammer` were not meant to be used like that.